src/CMakeLists.txt

```diff
@@ -100,6 +100,14 @@ include_directories(SYSTEM ${EIGEN3_INCLUDE_DIR})
 find_package(websocketpp REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system)
 
+# Find Frozen, library used for constexpr immutable containers
+find_package(frozen REQUIRED)
+# Frozen is just an interface library, only containing headers and some CMake settings. It
+# should be fine to "link" the library to every target, since if the headers are not included,
+# it will not have any effect on size or performance. This will help us not have to worry about
+# making sure the headers are available in every file that includes Constants.h, for example.
+link_libraries(frozen::frozen)
```

Comment:
Good call, I didn't know this existed! It would be very tedious to manually include the headers in all build targets that need it.